Makefile

```diff
@@ -40,9 +40,15 @@ show-docs:
 
 linkcheck:
 	uv pip install -e ".[docs]"
+	# this checks: mkdocs.yml is valid, all listed pages exist, plugins are correctly configured, no broken references in nav or Markdown links (internal), broken links and images (internal, not external)
 	mkdocs build --strict
-	vale docs
-	linkchecker site/
+	# lychee checks for broken external links in the built site
+	lychee --offline site/
```

Comment:
Should lychee be added as doc dependency?

Reply:
oh i don't think so as lychee is not a Python package 🤔

Reply:
I tried locally after `brew install lychee` and it works